tests/foreman/cli/test_errata.py

```diff
@@ -204,17 +192,17 @@ def test_positive_install_by_hc_id_and_org_id(self):
         """
         install_task = HostCollection.erratum_install(
             {
-                'id': self.host_collection['id'],
-                'organization-id': self.org['id'],
-                'errata': [self.CUSTOM_ERRATA_ID],
+                'id': host_collection['id'],
```

Comment:
Where does `host_collection` come from in this use?

Looks like the fixture called `setUp` needs to return some entities for the tests to use.

Reply:
I have reworked the fixtures, please review again